homeassistant/components/litterrobot/sensor.py

```diff
@@ -98,6 +98,24 @@ def icon(self) -> str | None:
             device_class="litterrobot__status_code",
             entity_category=EntityCategory.DIAGNOSTIC,
         ),
+        RobotSensorEntityDescription[LitterRobot](
+            key="is_sleeping",
+            name="Is sleeping",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            entity_registry_enabled_default=False,
+        ),
+        RobotSensorEntityDescription[LitterRobot](
+            key="sleep_mode_enabled",
+            name="Sleep mode enabled",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            entity_registry_enabled_default=False,
+        ),
+        RobotSensorEntityDescription[LitterRobot](
+            key="power_status",
+            name="Power status",
+            entity_category=EntityCategory.DIAGNOSTIC,
+            entity_registry_enabled_default=False,
+        ),
```

Comment:
The dependency has been bumped, so let's make sure this is available for all robots now.

Reply:
Is power status only either ac and battery? We could just make it binary then. What about the other 2?

Reply:
That field comes back AC, DC, or NC (not connected)

Reply:
Though you still could make it just a binary sensor to say when it is running on battery

Reply:
How's this? I think the `PLUG` device class is a good one.